### PR TITLE
Ports tg#54991 tail entwining

### PR DESCRIPTION
**Pull request by Blueturbo47**
Ports https://github.com/tgstation/tgstation/pull/54991

Very useful for roleplay, quick merge this.

### CODE REVIEW

**Comment by TeomanTheGreat** [comment]
die

**Comment by TeomanTheGreat** [comment]
also you merged from master

**Comment by optimumtact** [comment]
:+1: stops game breaking crashes due to misisng player immersion

**Comment by Kylerace** [comment]
why isnt this merged yet this is too important to wait

**Comment by TeomanTheGreat** [comment]
fuck off from our pr tg players

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -285,9 +285,13 @@
 
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER)))
-			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>", \
+			if(isfelinid(AM) && isfelinid(src))
+			M.visible_message("<span class='warning'>[src] coils their tail with [AM], wow is that okay in public?!</span>", "[src] has entwined their tail with yours!")
```
Comment (LemonInTheDark) [inline code/modules/mob/living/living.dm]: you ported our bugs too 
```suggestion
	            M.visible_message("<span class='warning'>[src] coils their tail with [AM], wow is that okay in public?!</span>", "[src] has entwined their tail with yours!")
```

**Comment by TeomanTheGreat** [comment]
he made the pr from master btw :flushed:

**Comment by TeomanTheGreat** [comment]
@qwertyquerty please close this bro

**Comment by Kylerace** [comment]
> @qwertyquerty please close this bro

we get it you dont like it you dont have to spam it

**Comment by TeomanTheGreat** [comment]
oh also did you know that oranges shadow reverted the pr on tg....

**Comment by ghost** [comment]
What up bitches! If /tg/ hadn’t permad me from github I would have to say this but speed close this ungodly abomination!

**Comment by LemonInTheDark** [comment]
> 
> 
> fuck off from our pr tg players

How dare you imply coders play

**Comment by TeomanTheGreat** [comment]
Ah yes how could I forget my bad

**Comment by ike709** [comment]
Wanted to merge this for the meme but it doesn't even compile.

**Comment by qwertyquerty** [comment]
http://static.qtqt.cf/videos/crapola/hilarious.mp4

**Pull request closed without merging**